scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -479,9 +479,12 @@ class FossIdTest : WordSpec({
 
             shouldThrow<TimeoutCancellationException> {
                 withTimeout(1000) {
-                    fossId.scanPackages(
-                        setOf(createPackage(createIdentifier(index = 1), vcsInfo)), emptyMap()
-                    )
+                    launch {
```

Comment:
Note to myself: I need to think about whether `launch` is the correct thing to use here.

Reply:
The reason to use `launch` here was that the code must be non-blocking for the timeout to work correctly.

Reply:
The surrounding `withTimeout(1000)` cancels the coroutine running inside it afer the given milliseconds and if it is still running it throws a `TimeoutCancellationException`. If it would be finished already, it would not throw an exception.